File: CommunicationSystem/ClientApp/src/app/messenger/userlastmessage.ts
export class UserLastMessage {
  constructor(
    public id: number,
    public nickName: string,
    public accountImage: string,
    public email: string,
    public messageId: number,
    public from: number,
    public to: number,
    public content: string,
    public notViewed: number,
    public date?: Date,
    public userActivity?:string
  ) { }
}

File: CommunicationSystem.Domain/Entities/UserLastMessage.cs

namespace CommunicationSystem.Domain.Entities
{
    public class UserLastMessage
    {
        public int Id { get; set; }
        public string NickName { get; set; }
        public string accountImage { get; set; }
        public string Email{get;set;}
        public long MessageId { get; set; }
        public int From { get; set; }
        public int To { get; set; }
        public string Content { get; set; }
        public DateTime? Date { get; set; }
        public int NotViewed { get; set; }
        public string UserActivity { get; set; }
    }
}
